packages/liveblocks-core/src/room.ts

```diff
@@ -2832,7 +2835,7 @@ export function createRoom<
     fetchClientApi
   );
 
-  async function fetchJson<T>(
+  async function fetchInboxNotificationJson<T>(
```

Comment:
@marcbouchenoire Calling it `fetchInboxNotificationJson` because it specifically throws `NotificationsApiError`. Do you have an alternative suggestion for the name?

Reply:
Since it's just `NotificationsApiError` we could call it `fetchNotificationsJson`?

Reply:
Yes! I like it over `fetchInboxNotificationJson` and done!